webservice/src/Core/IntelPage/Model/Pager.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Core\IntelPage\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use InvalidArgumentException;
+use Symfony\Component\Uid\Ulid;
+use function strlen;
+
+#[ORM\Entity]
+class Pager
+{
+    public const int PAGER_SLOT_MIN = 0;
+    public const int PAGER_SLOT_MAX = 7;
+
+    #[ORM\Id]
+    #[ORM\Column(type: 'ulid')]
+    private readonly Ulid $id;
+
+    /**
+     * @var string human identifier describing pager use
+     */
+    #[ORM\Column(length: 255)]
+    private string $label;
+
+    /**
+     * @var int human identifier as printed on the hardware
+     */
+    #[ORM\Column]
+    private int $number;
+
+    /**
+     * List of cap code assignments.
+     *
+     * @var Collection<int, AbstractCapAssignment>
+     */
+    #[ORM\OneToMany(
+        targetEntity: AbstractCapAssignment::class,
+        mappedBy: 'slots',
```

Comment:
```suggestion
        mappedBy: 'pager',
```

Mapped by should refer to the corresponding property on the Assignment class.